torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1147,15 +1150,26 @@ def summon_full_params(
             writeback (bool, Optional): if ``False``, modifications to params are
                 discarded after the context manager exists;
                 disabling this can be slightly more efficient (default: True)
+            rank0_only (bool, Optional): if ``True``, full parameters are
+                materialized on only global rank 0. This means that within the
+                context, only rank 0 will have full parameters and the other
+                ranks will have sharded parameters. This further means that for
+                non-zero ranks, the `writeback` argument is ignored and
```

Comment:
The comment is not complete?

Reply:
Instead of implicitly ignoring writeback, should we ask users to explicitly writeback to False?